core/src/main/java/org/jboss/pnc/rex/core/RemoteEntityClient.java

```diff
@@ -65,16 +65,26 @@ public void stopJob(Task task) {
         Request requestDefinition = task.getRemoteCancel();
 
         URI url;
+        org.jboss.pnc.api.dto.Request callbackRequest;
         try {
             url = new URI(requestDefinition.getUrl());
         } catch (URISyntaxException e) {
             throw new IllegalArgumentException("remoteCancel.url is not a valid URL for task with name " +
                     task.getName(), e);
         }
 
+        String callback = baseUrl + "/rest/internal/"+ task.getName() + "/finish";
+
+        try {
+            URI callbackUri = new URI(callback);
+            callbackRequest = new org.jboss.pnc.api.dto.Request(org.jboss.pnc.api.dto.Request.Method.POST, callbackUri);
```

Comment:
The service that calls back will have to add a Content-type

Reply:
Yup!